cnv/cnv_users_guide/cnv-editing-vm-template.adoc

```diff
@@ -6,6 +6,5 @@ toc::[]
 
 You can edit a virtual machine template in the web console.
 
-include::modules/cnv-editing-template-web.adoc[leveloffset=+1]
-
-
+include::modules/cnv-editing-vm-yaml-web.adoc[leveloffset=+1]
```

Comment:
This should be `cnv-editing-template-web.adoc. The functionality of `cnv-editiing-vm-yaml-web` is not available in 2.1

Reply:
Done.